install-agora-git.sh

```diff
@@ -37,63 +52,69 @@ gmake
 sudo gmake install
 cd ..
 rm -rf Build
-popd
+cd $AGORA_HOME
+
+sudo -E ldconfig
```

Comment:
this line may not be necessary. It was something I added when base's configure script couldn't find libobjc. The additional flags to make's configure scripts seemed to address the problem.